modules/installation-three-node-cluster-cloud-provider.adoc

```diff
@@ -38,16 +42,26 @@ Because application workloads run from control plane nodes, additional subscript
 
 .Procedure
 
+ifdef::nutanix[]
+* Set the number of compute replicas to `0` in your `install-config.yaml` file, as shown in the following `compute` stanza:
+endif::nutanix[]
+
+ifndef::nutanix[]
 . Set the number of compute replicas to `0` in your `install-config.yaml` file, as shown in the following `compute` stanza:
+endif::nutanix[]
 +
+.Example `install-config.yaml` file for a three-node cluster
 [source,yaml]
 ----
+apiVersion: v1
+baseDomain: example.com
 compute:
 - name: worker
   platform: {}
   replicas: 0
+# ...
```

Comment:
Is it possible to get the example to tuck under the numbered step as with the previous example?

Reply:
![image](https://github.com/openshift/openshift-docs/assets/25209155/ca0c813e-9399-4c4d-8850-2ab3a30dbf82)

Reply:
That second example is for a UPI installation. A Nutanix specific UPI installation is not supported.

Reply:
I was just wondering if you could line it up under the numbered step. Not a big deal....

Reply:
Oh. Sorry for the confusion @mburke5678. Was clearly overthinking your comment.